test/util.spec.js

```diff
@@ -45,21 +45,39 @@ describe('util', () => {
   })
 
   it('.serialize and .deserialize large objects', (done) => {
-    const size = 65536 * 2
+    // larger than the default borc heap size, should auto-grow the heap
+    const size = 1 << 17 // 128Kb
```

Comment:
Nitpick: As these tests are all about sizes I would find `dataSize` clearer than just `size` (there's another instance below). Again something you don't need to change, I just enjoy having those comments when I get reviews as naming things has lots to do with experience.